helm-charts/README.md

```diff
@@ -0,0 +1,46 @@
+# Aegis Helm Chart
+
+Aegis keeps your secrets secret. With Aegis, you can rest assured that your sensitive data is always secure and protected. Aegis is perfect for securely storing arbitrary configuration information at a central location and securely dispatching it to workloads.
+
+## Installation
+
+To use Aegis, follow the steps below:
+
+1. Add Aegis Helm repository:
+
+    ```bash
+    helm repo add aegis https://shieldworks.github.io/aegis/
+    ```
+
+2. Update helm repository:
+
+    ```bash
+    helm repo update
+    ```
+
+3. Install Aegis using Helm:
+
+    ```bash
+    helm install aegis aegis/aegis --version 0.1.0
+    ```
+
+## Options
+
+The following options can be passed to the `helm install` command to set global variables:
+
+- `--set global.deploySpire=<true/false>`: This flag can be passed to install or skip Spire.
```

Comment:
+1 for using camelCase for variable names; it’s the typical convention as far as I’ve observed in many helm charts.